toolkit/tools/internal/grub/tokentests/CommonLineCommandLine.test

```diff
@@ -0,0 +1 @@
+linux $bootprefix/$mariner_linux rd.auto=1 root=$rootdevice $mariner_cmdline lockdown=integrity sysctl.kernel.unprivileged_bpf_disabled=1 $systemd_cmdline console=tty0 console=ttyS0 $kernelopts
```

Comment:
suggestion - maybe add a test for a more involed kernel cmdline. ie
```
console=tty0 console=ttyS0 intel_iommu=on iommu=pt
      processor.max_cstate=1 intel_idle.max_cstate=0 msr.allow_writes=on
      default_hugepagesz=1G hugepagesz=1G hugepages=40 skew_tick=1 nohz=on
      nohz_full=2-31,34-63 rcu_nocbs=2-31,34-63 systemd.cpu_affinity=0,1,32,33
      rcu_nocb_poll nmi_watchdog=0 idle=poll kthread_cpus=0,1,32,33
      irqaffinity=0,1,32,33 isolcpus=nohz,domain,managed_irq,2-31,34-63
      tsc=reliable iomem=relaxed nosoftlockup net.ifnames=1 audit=0 rd.luks=0
```

Reply:
Fixed.